src/detect-engine-mpm.c

```diff
@@ -86,9 +86,9 @@ static int g_mpm_list_cnt[DETECT_BUFFER_MPM_TYPE_SIZE] = { 0, 0, 0 };
  *
  *  \note to be used at start up / registration only. Errors are fatal.
  */
-void DetectAppLayerMpmRegister(const char *name, int direction, int priority,
+static void DetectAppLayerMpmRegisterAux(const char *name, int direction, int priority,
```

Comment:
aux means "extra", I don't see how that makes sense here. I think it just acts like `Internal` or similar? We use other patterns like:
`DetectAppLayerMpmRegisterDo`
`DetectAppLayerMpmRegisterReal`
`DetectAppLayerMpmRegisterInternal`
Since this is a private function, we could also drop the name space prefix and just have `Register` or something.

Reply:
ok will do after https://github.com/OISF/suricata/pull/11028

Reply:
Oh, this is #11028 🤦